docs/app/docs/devbox_cloud/index.mdx

```diff
@@ -61,11 +59,11 @@ https://devbox.sh/app/projects/<projectId>/port/8080
 
 This URL redirects to `<hostname>-<port>.svc.devbox.sh`, which points to the actual host and port of your project's VM machine.
 
-### Opening in desktop VS Code
+### Opening in Desktop VS Code
 
 Click on the `Open in Desktop` button on the top right corner, which will open up your local VS Code editor. Follow the prompt, and you should see the VS Code window refreshes and connects to the remote machine via ssh.
 
-### SSH via local terminal
+### Access your Playground via SSH
```

Comment:
I think we should delete this section, because currently it does not actually work